plugins/woocommerce/tests/api-core-tests/playwright.config.js

```diff
@@ -0,0 +1,49 @@
+const { devices } = require( '@playwright/test' );
+require( 'dotenv' ).config();
+
+const base64auth = btoa(
```

Comment:
Since these tests will fail if you don't create the `.env` file in the `plugins/woocommerce` directory (which we likely won't want to create this here in the future anyway), so I had a minor suggestion. Let me know your thoughts on this @nigeljamesstevenson. I am thinking we can update the auth logic to be the following:

```javascript
let userKey = 'admin';
let userSecret = 'password';

if ( process.env.USER_KEY ) {
	userKey = process.env.USER_KEY;
}

if ( process.env.USER_SECRET ) {
	userSecret = process.env.USER_SECRET;
}

const base64auth = btoa( `${ userKey }:${ userSecret }` );
```

This way we can handle if one isn't set. Eventually we'll want to support a local `.env` file to the tests, but we can handle that as we approach preparing to release the package. Otherwise, if we don't do this change, we at least will want to document that you need to have that `.env` file set in the `plugins/woocommerce` directory otherwise the tests won't run. I lean towards adding in the conditional logic above as an initial fix then we can make it dynamic in a future PR.

I made this change locally and tested that it works both with the `.env` file and without it.

Reply:
This sounds good to me @zhongruige 👍

The original implementation was following the pattern of the existing supertest tests (and therefore using the `.env` file in the `plugins/woocommerce` directory) - I agree the logic can be updated (and make execution one step simpler for the user). 
I will also add a note regarding the use of an `.env` file for completion.. 
I also added in similar for the baseURL to keep it in line with the username and password.

In short - consider it done!

I also noticed that the test execution instructions in the readme didn't clearly state the command to kick off the tests so added an extra line in there and also referenced the fact that the destroy and create test env commands may also need to be executed in `plugins/woocommerce` (or a subdirectory thereof)

Reply:
Thank you for making these changes and also for updating the documentation, too!